src/main/java/com/ita/if103java/ims/controller/ItemController.java

```diff
@@ -0,0 +1,95 @@
+package com.ita.if103java.ims.controller;
+
+import com.ita.if103java.ims.dto.ItemDto;
+import com.ita.if103java.ims.dto.SavedItemDto;
+import com.ita.if103java.ims.dto.WarehouseDto;
+import com.ita.if103java.ims.service.ItemService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping({"/items"})
+public class ItemController {
+    private ItemService itemService;
+
+    @Autowired
+    public ItemController(ItemService itemService) {
+        this.itemService = itemService;
+    }
+    @PostMapping("/addItem")
+    @ResponseStatus(HttpStatus.OK)
+    public ItemDto addItem(@RequestBody ItemDto itemDto) {
+        return itemService.addItem(itemDto);
+    }
```

Comment:
just /add or even without any additional path.

Read information for url pattern in the REST